src/main/webapp/app/pages/teamPage/TeamPage.tsx

```diff
@@ -61,6 +61,13 @@ export const TeamPage = () => {
       faculty: false,
       showCOI: true,
     },
+    {
+      firstName: 'Stephanie',
```

Comment:
@scarrero4660 sorry, I forgot to update the website at the time of joining. Could you double check your info?

Reply:
No problem. My only note is that I have a BA not a BSc

Reply:
Sounds good, let me update.